src/FluentValidation.Tests/ValidatorTesterTester.cs

```diff
@@ -86,6 +91,18 @@ public class ValidatorTesterTester {
 			});
 		}
 
+		[Fact]
+		public void ShouldHaveValidationError_Identity_Should_not_throw_when_there_are_validation_errors() {
+			validator.ShouldHaveValidationErrorFor<Person>(x => x, new Person { Forename = "Helle", Surname = "Helle" }, "default");

```

Comment:
The generic type parameters for these should be automatically inferred, so they can be removed from the method call.